### PR TITLE
added improvements that appeared after code review

### PR DESCRIPTION
**Pull request by spetrychenko**

### CODE REVIEW

File: oauth-app/src/main/resources/application.yml [hunk 1]
```diff
@@ -18,23 +18,22 @@ server:
   port: 8082
 
 oauth:
+  server:
+    host:
```
Comment (artemsnisarenko) [inline oauth-app/src/main/resources/application.yml]: host/url discrepancy here and in the code

File: oauth-rest-server/src/main/java/de/adorsys/psd2/oauth/controller/AuthorizationController.java [hunk 2]
```diff
@@ -80,6 +84,11 @@ public RedirectView getAuthorizationCode(
         return redirectToErrorPage(errReq.error, errReq.error_description, errReq.error_uri, redirectAttributes);
     }
 
+    String buildRedirectUri(String bank) throws BankNotSupportedException {
```
Comment (artemsnisarenko) [inline oauth-rest-server/src/main/java/de/adorsys/psd2/oauth/controller/AuthorizationController.java]: Superficial throws declaration.

File: oauth-service-impl/src/main/java/de/adorsys/psd2/oauth/service/impl/TokenServiceImpl.java [hunk 3]
```diff
@@ -104,7 +97,7 @@ public TokenBO findById(String id) throws TokenNotFoundServiceException {
 
     @Override
     public TokenBO save(TokenBO token) {
-        logger.info("Trying to save token {}", token);
+        logger.info("Trying to save token with id={}", token.getId());
```
Comment (artemsnisarenko) [inline oauth-service-impl/src/main/java/de/adorsys/psd2/oauth/service/impl/TokenServiceImpl.java:100]: Is id initialized at this point?

File: oauth-app/src/main/resources/application.yml (hunk 1 above)
Comment (spetrychenko) [reply oauth-app/src/main/resources/application.yml]: fixed

File: oauth-rest-server/src/main/java/de/adorsys/psd2/oauth/controller/AuthorizationController.java (hunk 2 above)
Comment (spetrychenko) [reply oauth-rest-server/src/main/java/de/adorsys/psd2/oauth/controller/AuthorizationController.java]: removed throwing exception

File: oauth-service-impl/src/main/java/de/adorsys/psd2/oauth/service/impl/TokenServiceImpl.java (hunk 3 above)
Comment (spetrychenko) [reply oauth-service-impl/src/main/java/de/adorsys/psd2/oauth/service/impl/TokenServiceImpl.java:100]: it is present

**Pull request merged**